### ISSUE TITLE
Auto create dev users

### ISSUE THREAD

**Issue by amenk** [netz98/n98-magerun#779]
It would be nice to have a function to auto create dev users (one user per defined role) for example in the schema:

dev_administrators - password: dev123456 - example dev.administrators@example.com - Name: "Dev user - password is dev123456"

**Comment by cmuench** [comment]
@amenk That can be done by a simple magerun script which can be placed i.e. in a script repo.

```
admin:user:create user1 user1@example.com mypasswd Firstname Lastname myrole
admin:user:create user2 user2@example.com mypasswd Firstname Lastname anotherrole
```
https://github.com/netz98/n98-magerun#n98-magerun-script-repository

**Comment by amenk** [comment]
Yes that is what I am currently doing - but it includes updating the script for each new project / when adding roles. So it would be awesome to auto detect all roles and create dev users.
